stdlib/3/importlib/__init__.pyi

```diff
@@ -0,0 +1,23 @@
+import sys
+import types
+from typing import Any, Mapping, Optional, Sequence
+
+
```

Comment:
In stubs we usually only leave one blank line between definitions (zero between methods in a class).